resources/assets/less/bem/profile-extra.less

```diff
@@ -49,10 +49,38 @@
     font-size: 18px;
     font-weight: 500;
     margin: 0 0 (@spacing * 2);
+    display: table;
 
     &--small {
       font-size: 15px;
       margin-top: (@spacing * 3);
     }
   }
+
+  &__dragdrop-toggle {
+    font-size: 18px;
+    position: relative;
+    float: right;
+    cursor: move;
+  }
+
+  &__list {
```

Comment:
bem violation aside, using `osu-layout__row` instead of this (at same div) seems to do same thing?